strings/strings.c

```diff
@@ -10,11 +10,14 @@
 int string_length(char s[])
 {
     int length = 0;
-    for (int i = 0; i < sizeof(s); i = i + 1)
-    {
-        length = length + 1;
-    }
-    return length;
+    int array_size = *(&s + 1) - s;
+    printf("%d", array_size);
+    // int array_size = sizeof(s) / sizeof(s[0]);
+    // for (int i = 0; i < array_size; i = i + 1)
+    // {
+    //     length = length + 1;
+    // }
+    return array_size;
```

Comment:
What are you trying to do with line 13: `int array_size = *(&s + 1) - s;` ?

Reply:
i was trying to get the size of char array and divide by the array itself. i found this on stackoverflow but i didnt really understand it